theano/sandbox/gpuarray/elemwise.py

```diff
@@ -69,9 +86,11 @@ def __str__(self):
 
     def make_node(self, *inputs):
         res = Elemwise.make_node(self, *inputs)
+        inputs = [as_gpuarray_variable(i, context=self.context)
```

Comment:
I don't add the context in the op because there are multiple inputs, but because there is something else that requires it.  In this case it's NVCC_compiler.